README.md

```diff
@@ -138,7 +138,7 @@ If `true`, will enable [`Hot Module Replacement`](https://webpack.js.org/concept
 
 _Note: If the build process generates errors, the client (browser) will not be notified of new changes and no HMR will be performed. Errors must be resolved before HMR can proceed._
 
-_Note: If using in combination with `http2`, the `http2` option `allowHTTP1` must be enabled for the HMR WS connection to work._
+_Note: If using in combination with `http2`, the `http2` option `allowHTTP1` will enabled by default for the HMR WS connection to work._
```

Comment:
```suggestion
_Note: If using in combination with `http2`, the `http2` option `allowHTTP1` will be enabled by default for the HMR WS connection to work._
```

Reply:
So I can't get the integration tests to run to completion locally, particularly the multi-hmr one seems to be troublesome.  It does happen as well on the master branch, so I'm not quite sure what is going on there.

Reply:
Can you tell more about the test issue? How does it fail exactly?